### ISSUE TITLE
Feature req: ability to disable saving thumbnails

### ISSUE THREAD

**Issue by ntlgr** [JimStar/reolink_cctv#185]
Saving thumbnails for each camera every time there is motion is consuming resources. It wears out SSDs, increases the size of full backups, slows down uploads of backups to cloud, and consumes expensive cloud storage space.

I am sure I am not the only one wanting to disable it. I have 16 cameras and this is getting quite hard to manage.
Having to create an automation to delete thumbnails on a daily basis sounds only like a workaround.

Requesting the ability to completely disable this feature should someone want so.

Thank you for this great integration that otherwise works flawlessly.

**Comment by lobhater** [comment]
I'm really new to all this stuff but I'm curious what would stop you from writing 1 automation that deletes all the thumbnails everyday at midnight or whenever your personal preferred schedule is?

**Comment by JimStar** [comment]
> I'm really new to all this stuff but I'm curious what would stop you from writing 1 automation that deletes all the thumbnails everyday at midnight or whenever your personal preferred schedule is?

Even easier: just set this value to zero days. It will only store last day images.

<img width="594" alt="Screen Shot 2023-04-23 at 10 01 44" src="https://user-images.githubusercontent.com/3421219/233808400-040055c6-4b36-4831-aad2-5da21dc6a435.png">

**Comment by JimStar** [comment]
... ah, and of course after setting the above parameter, the periodic (once a day) automation like this still needs to be set up:

<img width="1062" alt="Screen Shot 2023-04-23 at 17 07 11" src="https://user-images.githubusercontent.com/3421219/233820943-5fea2679-e4b1-40ff-bcea-cd8fd58a3dea.png">

**Comment by ntlgr** [comment]
Thank you for the responses, as I mentioned I see this as a workaround style approach (Setting the value to 0 still keeps for 1 day, I see this logic as a bit flawed). I have already configured it like that and my thumbnails are around 600MB per day, hence the feature request. Anyway no issues, was just a suggestion. I have now set the target folder to /dev/null for one of the cameras to see if everything is still working as it should.
